indico/modules/events/abstracts/operations.py

```diff
@@ -62,3 +62,29 @@ def delete_abstract(abstract, delete_contrib=False):
     logger.info('Abstract %s deleted by %s', abstract, session.user)
     abstract.event_new.log(EventLogRealm.management, EventLogKind.negative, 'Abstracts',
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
+
+
+def schedule_cfa(event, data):
+    event.cfa.schedule(data)
+    logger.info("Call for abstracts for %s scheduled by %s", event, session.user)
+    date_format = '%d/%m/%Y %H:%M'
+    log_data = {
+        'Start': data['start_dt'].strftime(date_format),
```

Comment:
While you are right that it should not be done with strftime, this is data shown to humans so I don't think "code" format is the right choice. Maybe check if we put a date in log data anywhere else and see which format we use there?

Or maybe we could just use `.isoformat()`. That's human-readable but more important we could easily add some extra code in the future when displaying log data that checks with a whether it's an isoformat date string and then formats it in a pretty way.